app/models/address.rb

```diff
@@ -33,13 +33,13 @@ def to_a
   private
 
   def valid_uk_postcode
-    if postcode.present?
-      uk_postcode = UKPostcode.parse(postcode)
-      if uk_postcode.full_valid?
-        errors.add(:postcode, :geocoding_error) unless point.present?
-      else
-        errors.add(:postcode, :non_uk)
-      end
+    return if postcode.blank?
```

Comment:
This is also one of those rubocop inspired things I take it? Not sure it should be in this PR / commit.